### ISSUE TITLE
registerDevice callbacks aren't called when push notifications are disabled on iOS

### ISSUE THREAD

**Issue by lietu** [Pushwoosh/pushwoosh-phonegap-plugin#141]
If a user installs the app on iOS, and does not allow push notifications the `registerDevice()` -callbacks do not get called.

```javascript
document.addEventListener('deviceready', function() {
    var plugin = cordova.require('com.pushwoosh.plugins.pushwoosh.PushNotification');

    plugin.onDeviceReady({
        pw_appid: "my-app-id"
    });

    plugin.registerDevice(
        function() { console.log("OK"); },
        function() { console.log("NOT OK"); }
    );
});
```

If push notifications are not allowed (either during first run, or later from system settings), neither callback gets executed.

If they are allowed it prints `OK` with the correct configuration.

I was not able to reproduce on Android by disabling notifications from the app from settings.

We can use `getRemoteNotificationStatus()` to check if they are not allowed and skip the `registerDevice` -calls for now, but it would be a lot better to have the error callback be triggered.

**Comment by acti0nm4n** [comment]
+1 
A working callback would be nice!

**Comment by RonnySchleicher** [comment]
I have the same problem.
I had also expected that the error callback is called. 
What happens with this Issue?
Status is open but there is no response?

**Comment by DimanAM** [comment]
Fixed in 6.0.0
